gnuradio-runtime/lib/basic_block.cc

```diff
@@ -201,7 +202,8 @@ namespace gr {
     gr::thread::scoped_lock guard(mutex);
 
     if((msg_queue.find(which_port) == msg_queue.end()) || (msg_queue_ready.find(which_port) == msg_queue_ready.end())) {
-      std::cout << "target port = " << pmt::symbol_to_string(which_port) << std::endl;
+      GR_LOG_DEBUG (d_logger, "target port = " << pmt::symbol_to_string(which_port));
```

Comment:
Since this is an error condition, this should be GR_LOG_ERROR. Also, there is no need to duplicate all the logging calls to both loggers; the debug ones should be going to d_debug_logger only, the rest to d_logger.